bciers/libs/components/src/datagrid/cells/operations/OperationsActionCell.tsx

```diff
@@ -1,11 +1,10 @@
 import { GridRenderCellParams } from "@mui/x-data-grid";
 import { OperationStatus } from "@bciers/utils/src/enums";
-import Link from "next/link";
 
 const OperationsActionCell = (isInternalUser: boolean) => {
   const renderCell = (params: GridRenderCellParams) => {
     let actionText = "View Operation";
-    let url = `/operations/${params.row.id}?operations_title=${params.row.name}`;
+    let url = `../administration/operations/${params.row.id}?operations_title=${params.row.name}`;
```

Comment:
@BCerki 
**Suggestion:** use absolute paths (/path) rather than upward path (../path) 

I believe the preferred pattern of navigating to a route(app) relative to the root of the current zone is to use absolute paths (/path) - as we do in the dashboard fixture links.

I think the use of upward path (../path) will only work within the current app's directory structure; so, if we ever want to route outside of bciers  directory structure this pattern would fail.